src/js/util.js

```diff
@@ -1,30 +1,40 @@
 'use strict';
 
-fruskac.Util = (function () {
+fruskac.Util = (function() {
```

Comment:
We should have a "space" between "function" and "()" for anonymous functions, named functions should not have this space